vllm/worker/model_runner.py

```diff
@@ -277,7 +277,8 @@ def add_seq_group(self, seq_group_metadata: SequenceGroupMetadata):
             prefix_cache_hit = (computed_block_nums is not None
                                 and len(computed_block_nums) > 0
                                 and self.sliding_window is None and is_prompt)
-            if self.chunked_prefill_enabled and prefix_cache_hit:
+            if (self.chunked_prefill_enabled and prefix_cache_hit
+                    and self.attn_backend.get_name() != "flash-attn"):
                 raise RuntimeError(
                     "chunked prefill cannot be used with prefix caching now.")
```

Comment:
Update the error message that it is only suppored by flash attn backend (also please write how to enable flash attn backend. I.e., `VLLM_ATTENTION_BACKEND=FLASH_ATTN`)